packages/ui/.storybook/components/ColorPalette.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import { useTheme } from "@chakra-ui/react";
+import { ColorPalette as StorybookCollorPalette, ColorItem } from '@storybook/addon-docs/blocks';
+import { ThemeProvider } from '../../src/theme';
+
+interface ColorPaletteProps {
+  color: string;
+}
+
+export const AllVariants = ({color}: ColorPaletteProps) => {
+  const theme = useTheme();
+  
+  return <ColorItem title={`${color}`} subtitle='' colors={theme.colors[color]} />
+};
+
+export const SingleVariant = ({color}: ColorPaletteProps) => {
+  const theme = useTheme();
+  const pickedColor = theme.colors[color];
+
+  return <>{
+    Object.keys(pickedColor).map((range) => (
+      <ColorItem title={`${color}.${range}`} subtitle={pickedColor[range]} colors={[pickedColor[range]]} />
+    ))
+  }</>
+}
```

Comment:
🙃 There is no need to export things that wouldn't be imported anywhere

Reply:
You're right. 😃